### PR TITLE
i18n: Update German translation

### PR DESCRIPTION
**Pull request by palob**
A few fixes.

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -79,15 +80,15 @@ msgstr "Pixel"
...
+msgstr "Formatvorlage:"
 
 #: photocollage/gtkgui.py:686
 msgid "Border"
 msgstr "Rahmen"
 
 #: photocollage/gtkgui.py:691
 msgid "Thickness:"
-msgstr "Breite:"
+msgstr "Breite (relativ):"
```
Comment (adrienverge) [inline po/de.po]: Is "`relativ`" really needed, given that there's a "`%`" after the input field?

**Comment by adrienverge** [comment]
Danke schön @palob!

I just have a question before I merge this.

**Comment by palob** [comment]
Bitteschön.

I thought that ``relativ`` could slightly clarify what the ``%`` relates to.

You're right however, it's by no means crucial and might not look paticularly nice.

Shall I toss it?

**Comment by adrienverge** [comment]
I'm fine with both :+1:

**Comment by palob** [comment]
Alright, got rid of it :+1:

**Comment by adrienverge** [comment]
Thanks @palob, merging!

**Pull request merged**